examples/iot-nirvana/README.md

```diff
@@ -86,20 +102,20 @@ root of the project with the following parameters:
 
 Example:
 
-`run_oncloud.sh my-project my-bucket my-topic my_dataset.my_table`
+`run_oncloud.sh $PROJECT_ID $PROJECT_ID $PUBSUB_TOPIC $BIGQUERY_DATASET.$BIGQUERY_TABLE`
```

Comment:
I would suggest creating an environment variable $BUCKET_NAME in the same place as $PROJECT_ID in order to avoid all risk of confusion, even if they have the same name.